src/Item.php

```diff
@@ -647,7 +647,7 @@ public function get_copyright()
      * @param string $date_format Supports any PHP date format from {@see http://php.net/date} (empty for the raw data)
      * @return int|string|null
      */
-    public function get_date($date_format = 'j F Y, g:i a')
+    public function get_date(string $date_format = 'j F Y, g:i a')
```

Comment:
Apparently, this [accepted false](https://github.com/simplepie/simplepie/commit/4dac51faf47780847f32820b24c4b573247f9ca4) in the past but with the typehint it will still work as before since without strict type, it will be autocast. But we can drop the two casts below.

Reply:
The removal for `false` support was made in v1.3 which was technically a breaking change. But now it is documented and I would argue that adding the typehint would be ok.

Reply:
I removed the autocasting.